### PR TITLE
pd: units of the interval record changed form seconds to nanoseconds

### PR DESCRIPTION
**Pull request by nolouch**
Use more precision units for calculating the region heartbeat latency.

### CODE REVIEW

File: src/util/time.rs [hunk 1]
```diff
@@ -45,11 +45,9 @@ pub fn duration_to_nanos(d: Duration) -> u64 {
 
 /// Get the current timestamp in seconds.
```
Comment (overvenus) [inline src/util/time.rs]: Stale comments.

File: src/util/time.rs [hunk 2]
```diff
@@ -45,11 +45,9 @@ pub fn duration_to_nanos(d: Duration) -> u64 {
...
 #[inline]
-pub fn time_now_sec() -> u64 {
-    SystemTime::now()
-        .duration_since(UNIX_EPOCH)
-        .unwrap()
-        .as_secs()
+pub fn time_now_as_nanos() -> u64 {
+    let since_unix_epoch = SystemTime::now().duration_since(UNIX_EPOCH).unwrap();
+    since_unix_epoch.as_secs() * 1_000_000_000 + u64::from(since_unix_epoch.subsec_nanos())
```
Comment (overvenus) [inline src/util/time.rs]: How about `ms` or `us`? I don't think it needs the highest resolution. Is it breaking change?

**Comment by disksing** [comment]
I think second is enough. The clock time is not the same on different machines.

**Comment by nolouch** [comment]
we need another way to records the heartbeat latency. close this.

**Pull request closed without merging**